backends/xnnpack/operators/node_visitor.py

```diff
@@ -433,6 +442,14 @@ def define_tensor(
             else:
                 assert f"Unsupported weight per channel quantization axis for depthwise conv2d: {quant_params.axis}, expecting 0."
 
+        if swap_in_out_for_transpose_weights and (
```

Comment:
Nit: can we cleanly merge with depthwise clause above, OK if it gets harder to read.